packages/stats/gbstats/bayesian/tests.py

```diff
@@ -210,18 +220,24 @@ def compute_result(self) -> BayesianTestResult:
 class GaussianBayesianABTest(BayesianABTest):
     def __init__(
         self,
-        stat_a: Union[SampleMeanStatistic, RatioStatistic],
-        stat_b: Union[SampleMeanStatistic, RatioStatistic],
+        stat_a: Union[
+            SampleMeanStatistic,
+            RatioStatistic,
+            QuantileStatistic,
+            QuantileStatisticClustered,
```

Comment:
Does this class need to take QuantileStatistics? I don't think we ever want to pass it these, right?

Reply:
good catch,  I modified it as this was the first way I implemented Bayesian quantile approach, and I forgot to remove QuantileStatistics later.

Reply:
right, i updated after our convo